fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -659,6 +660,9 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
     @VariableMgr.VarAttr(name = HIVE_PARTITION_STATS_SAMPLE_SIZE)
     private int hivePartitionStatsSampleSize = 3000;
 
+    @VariableMgr.VarAttr(name = PIPELINE_SINK_DOP)
```

Comment:
we need to consider that the behavior of the system will not change after the user upgrades